validator/src/main/java/com/amazon/aoc/services/XRayService.java

```diff
@@ -61,7 +61,8 @@ public List<TraceSummary> searchClientCallTraces(String serviceName) {
                 .withFilterExpression(
                     "annotation.aws_local_service = \""
                         + serviceName
-                        + "\" AND annotation.aws_local_service = \"local-root-client-call\""));
+                        + "\" AND ( annotation.aws_local_service = \"local-root-client-call\" "
+                            + "OR annotation.aws_local_service = \"local-root-client-call:80\")"));
```

Comment:
Why does yours have port?

Reply:
I believe it's dotnet otel specific stuff, it's acceptable and still in the spec